src/docs/installation.md

```diff
@@ -59,10 +61,10 @@ At this point you will need to manually change the `app.json` that has been gene
     ],
     "version": "1.0.0",
     ...
-    "packagerOpts": {
++   "packagerOpts": {
```

Comment:
This change no longer needs to be made manually, since it is managed by vue-native-cli